styles/svg.ts

```diff
@@ -30,6 +34,52 @@ svg {
 .lastfm-user:hover {
     color: #f5222d;
 }
+.lastfm-profile {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    column-gap: 5px;
+}
+.lastfm-stats > div {
+    display: flex;
+    flex-direction: column;
+    row-gap:5px;
+}
+.lastfm-stats.compact > div {
+    display: flex;
+    flex-direction: row;
+    column-gap: 3px;
+    align-items: baseline;
+}
+.lastfm-stats > div > span:nth-child(2) {
```

Comment:
The font size is a bit too small for my liking, let's bump it to around 11-12px? It'll affect the height calculations I think.

Reply:
Bumped it up to 11px. I think 12px threw off the balance slightly, but in retrospect I don't think I tried changing the size of the numbers too - should look fine if it needs to be any larger.